apim-operator/pkg/controller/api/api_controller.go

```diff
@@ -2132,3 +2098,54 @@ func deleteCompletedJobs(namespace string) error {
 	}
 	return nil
 }
+
+func updateConfMapWithOwner(r *ReconcileAPI, owner []metav1.OwnerReference, configMap *corev1.ConfigMap) error {
+	configMap.OwnerReferences = owner
+	errorUpdateinterceptConf := r.client.Update(context.TODO(), configMap)
+	if errorUpdateinterceptConf != nil {
+		return errorUpdateinterceptConf
+	}
+	return nil
+}
+
+func interceptorHandler(r *ReconcileAPI, instance *wso2v1alpha1.API, owner []metav1.OwnerReference,
+	jobVolumeMount []corev1.VolumeMount, jobVolume []corev1.Volume, userNameSpace string) (bool, []corev1.VolumeMount, []corev1.Volume, error) {
+
```

Comment:
There are multiple places using instance.Name+"-interceptors". It is better if we move the "-interceptors" into a constant. And use the combined name in a single place and use it in the other places rather than repeating instance.Name+"-interceptors".

Reply:
It should be modified to take from the instance specification after updating CLI.

Reply:
I didn't get it :). What I meant is something like foo := instance + "-interceptor".
And then use **foo** across the necessary places

Reply:
Ignore the above comment. I got what you did there.

Reply:
Yeah. But here the name of the configmap should refer from the API instance specification when we keeping multiple copies. therefore need to get the configmap name from the instance specification.